logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -80,6 +80,40 @@ class MLSConversationDataSource(
             establishMLSGroup(groupID, members)
         }
 
+    override suspend fun addMemberToMLSGroup(groupID: String, members: List<UserId>): Either<CoreFailure, Unit> =
+        //TODO: check for federated and non-federated members
+        keyPackageRepository.claimKeyPackages(members).flatMap { keyPackages ->
```

Comment:
do we need to check if we already have key packages stored for the users?

Reply:
Actually we need to claim a new one for a new conversation, so we have to claim a new one, usually we always claim only one for that conversation.